docs/standard/generics/math.md

```diff
@@ -0,0 +1,199 @@
+---
+description: "Learn more about interfaces for generic math in .NET."
+title: Generic math
+titleSuffix: .NET
+ms.date: 07/26/2022
+---
+# Generic math
+
+.NET 7 introduces new math-related generic interfaces to the base class library. The availability of these interfaces means you can constrain a type parameter of a generic type or method to be "number-like". In addition, C# 11 and later lets you define [`static abstract` interface members](../../csharp/language-reference/keywords/interface.md#static-abstract-and-virtual-members). Because operators must be declared as `static`, this new C# feature lets operators be declared in the new interfaces for number-like types.
+
+Together, these innovations allow you to perform mathematical operations generically&mdash;that is, without having to know the exact type you're working with. For example, if you wanted to write a method that adds two numbers, previously you had to add an overload of the method for each type (for example, `static int Add(int first, int second)` and `static float Add(float first, float second)`). Now you can write a single, generic method, where the type parameter is constrained to be a number-like type. For example:
+
+```csharp
+static T Add<T>(T left, T right)
+    where T : INumber<T>
+{
+    return left + right;
+}
+```
+
+In this method, the type parameter `T` is constrained to be a type that implements the new <xref:System.Numerics.INumber%601> interface. <xref:System.Numerics.INumber%601> implements the <xref:System.Numerics.IAdditionOperators%603> interface, which contains the [+ operator](xref:System.Numerics.IAdditionOperators%603.op_Addition(%600,%601)). That allows the method to generically add the two numbers. The method can be used with any of .NET's numeric types, because they've all been updated to implement <xref:System.Numerics.INumber%601> in .NET 7.
```

Comment:
```suggestion
In this method, the type parameter `T` is constrained to be a type that implements the new <xref:System.Numerics.INumber%601> interface. <xref:System.Numerics.INumber%601> implements the <xref:System.Numerics.IAdditionOperators%603> interface, which contains the [+ operator](xref:System.Numerics.IAdditionOperators%603.op_Addition(%600,%601)). That allows the method to generically add the two numbers. The method can be used with any of .NET's built-in numeric types, because they've all been updated to implement <xref:System.Numerics.INumber%601> in .NET 7.
```

I think clarifying "built-in" is worthwhile as users may expect some form of "shape" matching against any .NET type instead.